Krishna Rajkumar Dembla/style.css

```diff
@@ -0,0 +1,726 @@
+/* IMPORTING FONTS FROM GOOGLE */
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Ubuntu:wght@400;500;700&display=swap');
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+}
+
+html {
+    scroll-behavior: smooth;
+}
+
+
+/* CUSTOM SCROLLL BAR */
+
+::-webkit-scrollbar {
+    width: 10px;
+}
+
+::-webkit-scrollbar-track {
+    background: #f1f1f1;
```

Comment:
Please make a dedicated function in which all the colors are stored and use that in your code. Also, make sure to use RGB values instead of hex values. You can make use of "Color Picker" if you want to see different types of values.

Reply:
The different functions used here are for different sections of the scroll bar.

Reply:
Hint: Make use of :root function